database/migrations/002_base_schema.up.sql

```diff
@@ -0,0 +1,244 @@
+-- rh_account
+CREATE TABLE IF NOT EXISTS rh_account
+(
+    id   INT GENERATED BY DEFAULT AS IDENTITY,
+    name TEXT NOT NULL UNIQUE,
+    CHECK (NOT empty(name)),
+    PRIMARY KEY (id)
+) TABLESPACE pg_default;
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON rh_account TO listener;
+
+-- system_platform
+CREATE TABLE IF NOT EXISTS system_platform
+(
+    id                       INT GENERATED BY DEFAULT AS IDENTITY,
+    inventory_id             TEXT                     NOT NULL,
+    CHECK (NOT empty(inventory_id)),
+    rh_account_id            INT                      NOT NULL,
+    first_reported           TIMESTAMP WITH TIME ZONE NOT NULL,
+    vmaas_json               TEXT,
```

Comment:
If this is guaranteed to be json, would `JSONB` work better than `TEXT`?

Reply:
There were some discussions about this. But I think then the checksum would not work, since JSONB might reformat the query. 

So we either remove the checksum or utilize JSONB